coordinator/clientapi/clientapi.go

```diff
@@ -558,6 +558,10 @@ func (a *ClientAPI) UpdateManifest(ctx context.Context, rawUpdateManifest []byte
 			return nil, 0, errors.New("recovery keys cannot be updated")
 		}
 	}
+	if currentManifest.Config.RecoveryThreshold != updateManifest.Config.RecoveryThreshold {
+		a.log.Error("UpdateManifest: Invalid manifest: Recovery threshold cannot be updated")
+		return nil, 0, errors.New("recovery threshold cannot be updated")
```

Comment:
just to be sure: we will be able to allow updating this together with "Allow updating recovery public keys on manifest update", right?